frontend/src/components/maps/BuildingPopupView.tsx

```diff
@@ -37,12 +38,17 @@ export const BuildingPopupView: React.FC<IBuildingPopupViewProps> = (
   const defaultZoom = () =>
     leaflet.map?.flyTo(
       [buildingDetail!.latitude as number, buildingDetail!.longitude as number],
-      14,
+      MAX_ZOOM,
+    );
+
+  const jumpToView = () =>
+    leaflet.map?.setView(
+      [buildingDetail!.latitude as number, buildingDetail!.longitude as number],
+      MAX_ZOOM,
```

Comment:
I used a different function here that doesn't have an animation because the combination of the side bar sliding out and the map zooming in was causing everything to slow down and look terrible